mozilla/foundation.mozilla.org
# removed company contact information fields from template and cms/models

Closes #7384

**Link to sample test page**: https://foundation-s-7384-remov-tgva3i.herokuapp.com/en/privacynotincluded/happen-during-key-key-court/

**Steps to test**: 

1. Visit the page linked above
2. Scroll down to where the company contact information used to be (underneath security), note that it is no longer there
3. If everything still looks ok with the section removed, testing is complete!
